### PR TITLE
Add BasicRelay tests

### PR DESCRIPTION
**Pull request by olivdb**

### CODE REVIEW

File: test/wormhole.test.ts [hunk 1]
```diff
@@ -452,6 +463,123 @@ export function runWormholeTests(domain: string, setupDomain: DomainSetupFunctio
       })
     })
 
+    describe('relay', () => {
+      let relay: BasicRelay
+      const expiry = '10000000000'
+      const gasFee = parseEther('1.0')
+
+      before(async () => {
```
Comment (krzkaczor) [inline test/wormhole.test.ts:471]: I'd use `beforeEach` blocks and drop finally clauses in test cases

Comment (olivdb) [reply test/wormhole.test.ts:471]: In general I agree but in this case the Rinkeby test setup is painfully slow and expensive (and the main Rinkeby faucet has been offline for weeks). So until Arbitrum supports running a local blockchain (or until the Rinkeby faucet is back online) I'd suggest to keep this structure.

Comment (krzkaczor) [reply test/wormhole.test.ts:471]: ahh okay... Can you perhpaps drop a comment why we do things this way then?

Comment (olivdb) [reply test/wormhole.test.ts:471]: yes, sure

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**